### ISSUE TITLE
not found error not working for `rocky` and other distros

### ISSUE THREAD

**Issue by vclemenzi** [quickemu-project/quickemu#971]
#### Expected behaviour

The typical error message when a distro does not exist

#### Actual behaviour

An error.

```bash
$ quickget rocky

ERROR! You must specify a release.
 - Releases: /usr/bin/quickget: line 2851: releases_rocky: command not found
 ```

#### Steps to reproduce the behaviour

Execute the command

#### Linux Distribution & Kernel

Run the following and paste the output.

```bash
LSB Version:    n/a
Distributor ID: EndeavourOS
Description:    EndeavourOS Linux
Release:        rolling
Codename:       rolling
Linux vc 6.7.6-arch1-1
```

**Comment by vclemenzi** [comment]
After a bit of research I find that this problem repeats with archlinux as well:

```bash
$ quickget arch
ERROR! You must specify a release.
 - Releases: /usr/bin/quickget: line 2851: releases_arch: command not found
```

**Comment by lj3954** [comment]
I also had found this issue and fixed it as a part of my PR #886.

**Comment by flexiondotorg** [comment]
Closed via #986
